qa/rpc-tests/test_framework/mininode.py

```diff
@@ -1614,7 +1615,7 @@ class NodeConn(asyncore.dispatcher):
         "regtest": b"\xfa\xbf\xb5\xda",   # regtest
     }
 
-    def __init__(self, dstaddr, dstport, rpc, callback, net="regtest", services=NODE_NETWORK):
+    def __init__(self, dstaddr, dstport, rpc, callback, net="regtest", services=NODE_NETWORK, send_version=True):
```

Comment:
Well, if you change a function name and everything fails suddenly, that's a pretty good indicator that you should change it back :)

Meh - you have a preference for named arguments, I have a slight preference for overriding the function. I'm not going to fight you on it.

But I would really like you to include the module docstring and addition comments from https://github.com/jnewbery/bitcoin/commit/9457e23996bf5169dc9841eeca6f4ba010ae8654, or equivalent.